Lightning not compiling/linking on Mac Mini M1

### Issue and Steps to Reproduce
After installing all dependencies and executing 
./configure
make

following error is generated:

ld ccan/ccan/cdump/tools/cdump-enumstr
ld: library not found for -lgmp
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [ccan/ccan/cdump/tools/cdump-enumstr] Error 1

### `getinfo` output

Same error here on my Mac Mini M1. 

ld ccan/ccan/cdump/tools/cdump-enumstr
ld: library not found for -lgmp
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [ccan/ccan/cdump/tools/cdump-enumstr] Error 1

@Quenos Have you found a solution?

--- a/doc/INSTALL.md
+++ b/doc/INSTALL.md
@@ -226,6 +226,11 @@ If you need SQLite (or get a SQLite mismatch build error):
     $ export LDFLAGS="-L/usr/local/opt/sqlite/lib"
     $ export CPPFLAGS="-I/usr/local/opt/sqlite/include"
 
+Some library paths are different when using `homebrew` with M1 macs, therefore the following two variables need to be set for M1 machines
+
+    $ export CPATH=/opt/homebrew/include
+    $ export LIBRARY_PATH=/opt/homebrew/lib
+
 If you need Python 3.x for mako (or get a mako build error):
 
     $ brew install pyenv

